pkg/banneduser/banneduser.go

```diff
@@ -0,0 +1,55 @@
+package banneduser
+
+import (
+	"context"
+	"fmt"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const bannedByLabel = toolchainv1alpha1.LabelKeyPrefix + "banned-by"
```

Comment:
Yes 🙂

Reply:
You can do a replace of the API  here in go.mod of toolchain common(go mod edit -replace) , and then once API pr merges you can remove the replace 
example- https://github.com/codeready-toolchain/host-operator/pull/1058/files 
this will help you checking real time if the changes in 1 repo has affect on other repos or not (before merging)

Reply:
Thank you so much!! Goinggg